server/chat-plugins/mafia.ts

```diff
@@ -144,6 +144,70 @@ function writeFile(path: string, data: AnyObject) {
 	));
 }
 
+function mafiaSearch(
+	entries: [string, MafiaDataAlignment | MafiaDataRole | MafiaDataTheme | MafiaDataIDEA | MafiaDataTerm][],
+	searchTarget: string, searchType: keyof MafiaData
+) {
+	if (typeof (entries) === 'undefined' || searchType === `aliases` || searchTarget.length === 0) return entries;
+
+	// Handle negation
+	const negation = searchTarget.startsWith('!');
+	if (negation) searchTarget = searchTarget.substring(1).trim();
+
+	const entriesCopy = entries.slice();
+
+	// Check if the search term is an alias of something
+	const alias = toID((toID(searchTarget) in MafiaData[`aliases`]) ?
+		MafiaData[`aliases`][toID(searchTarget)] : searchTarget);
+
+	if (searchType === `themes` && searchTarget.includes(`players`) && searchTarget.includes(`pl`)) {
+		// Search themes by playercount
+		const inequalities = ['<=', '>=', '=', '<', '>'];
+		const inequality = inequalities.find(x => searchTarget.includes(x));
+		if (!inequality) return entries; // this.errorReply(`Please provide a valid inequality for the players.`);
+
+		const players = searchTarget.split(inequality)[1].trim();
+		if (((players !== null) &&
+			(players !== '') &&
+			!isNaN(Number(players)))) {
+			if (inequality === '=') {
+				entries = entries.filter(([key]) => players in MafiaData[`themes`][key]);
+			} else if (inequality === '<' || inequality === '<=') {
+				entries = entries.filter(([key]) => ([...Array(+players + (inequality === '<=' ? +1 : +0)).keys()])
+					.some(playerCount => playerCount in (MafiaData[`themes`][key])));
```

Comment:
Why `+players`?

I recommend adding one or more comments here to explain whats going on to increase code readability and make maintenance easier.

Reply:
+players ensures that it is read as a number, not a character

Adding this as a comment, as well